app/src/main/java/agency/five/codebase/android/movieapp/ui/component/CrewItem.kt

```diff
@@ -0,0 +1,50 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.ui.theme.MovieAppTheme
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+data class CrewItemViewState(
+    val name: String,
+    val profession: String
+)
+
+@Composable
+fun CrewItem(
+    crewItemViewState: CrewItemViewState,
+    modifier: Modifier = Modifier
+) {
+    Column {
```

Comment:
Here you should add `modifier` which is passed as a parameter. Passed modifier should be used for top layouts in composable functions, like `Column` here, and every other component inside that columns should have a new `Modifier`